### DISCUSSION TITLE
kindling-collector如何识别是MySQL的协议？

### DISCUSSION THREAD

**Discussion by zcl1115** [harmonycloud/kindling#588] [agent collector] [answered]
我在我们团队的机器上部署了kindling-collector，然后在部分机器上采集到了entity的MySQL协议的数据，但是经排查这些机器上面都是没有部署MySQL的，想问一下是怎么识别的，通过前缀sql匹配吗

**Comment by dxsup** [comment, marked as answer]
首先默认将端口3306识别为MySQL协议，其次根据网络请求中的协议内容进行解析匹配。可以把数据发出来看一下
